docs/source/conf.py

```diff
@@ -24,7 +24,7 @@
 author = 'Julian Burella Perez'
 
 # The full version, including alpha/beta/rc tags
-release = '0.0.1'
```

Comment:
Why was this 0.0.1?

Reply:
Because it was never updated I think,
I think that previously we retrieved this value by importing `gph`, the issue with this is that you need to install `gph` to build the documentation. And, in my opinion, we should not do that.

Not for this release, but for a future release I will try to make this clever, and not hard-code the value at 2 different places.

Reply:
We have a similar issue in `giotto-tda`!